book/src/rent.md

```diff
@@ -0,0 +1,52 @@
+# Rent
+
+Accounts on Solana may have owner-controlled state (`Account::data`) that's separate 
+from the account's balance (`Account::lamports`).  Since validators on the network need 
+to maintain a working copy of this state in memory, the network charges a time-and-space 
+based fee for this resource consumption, also known as Rent.
+
+## Two-tiered rent regime
+
+Accounts which maintain a minimum balance equivalent to 2 years of rent payments are exempt.  Accounts whose balance falls below this threshold are charged rent at a rate specified in genesis, in lamports per kilobyte-year.  The network charges rent on a per-epoch 
+basis, in credit for the next epoch (but in arrears when necessary), and `Account::rent_epoch` keeps track of the next time 
+rent should be collected from the account.  
+
+## Collecting rent
+
+Rent is due at account creation time for one epoch's worth of time, and the new account has `Account::rent_epoch` of `current_epoch + 1`.  After that, the bank deducts rent from accounts during normal transaction processing as part of the load phase.
+
+If the account is in the exempt regime, `Account::rent_epoch` is simply pushed to `current_epoch + 1`.
+
+If the account is non-exempt, the difference between the next epoch and `Account::rent_epoch` is used to calculate the amount of rent owed by this account (via `Rent::due()`).  Any fractional lamports of the calculation are truncated.  Rent due is deducted from `Account::lamports` and `Account::rent_epoch` is updated to the next epoch.  If the amount of rent due is less than one lamport, no changes are made to the account.
+
+Accounts whose balance is insufficient to satisfy the rent that would be due simply fail to load.
+
+A percentage of the rent collected is destroyed.  The rest is distributed to validator accounts by stake weight, a la transaction fees, at the end of every slot.
+
+## Credit only
+
+Credit only accounts are treated as a special case.  They are loaded as if rent were due, but updates to their state may be delayed until the end of the slot, when credits are paid.
```

Comment:
Whats an example of a use-case for a credit only account?

Reply:
Crowdfunding

Reply:
CO accounts can be loaded by multiple transactions in a single batch to execute in parallel, because they don’t introduce a data dependency

Reply:
got it, thanks